docs/data-storage.md

```diff
@@ -1,80 +1,38 @@
-# Using Local Storage/Cookies
-
-Saving data must go through the [GMI](working-with-gmi.md), 
-this ensures all data access complies with BBC data policies.
-
-## Cookies/Local Storage disabled
-
-In the event a user has disabled cookies/local storage, any attempt to 
-save data will be ignored and any attempt to retrieve data will 
-return the global settings with default values.
-
-Games also adhere to the BBC 'Functional' Cookie so a user can opt out of these features. 
-Users can manage preferences here: [Manage Cookie Settings (Live)].
-
-*All games must be prepared to handle this scenario.*
+# Using Local Storage
 
 ## Saving data
 
-````
-gmi.setGameData(key, value);
-````
+Ideally, data should not be stored. If it is stored, data must not be personal and should adhere to the cookie policy of the browser. If the game contains sign in, it must respect the privacy settings on the user's account.
```

Comment:
Can get rid of this bit now.

Reply:
I was asked to keep this section with some changes in:
https://github.com/bbc/genie-starter-pack/pull/4